app/assets/stylesheets/home.sass

```diff
@@ -466,6 +466,15 @@ p
 .api-section-top-margin
   margin-top: 12%
 
+.flashes
+  text-align: center
+  @media (max-width: $breakpoint-tablet)
+    padding-left: 85px
+    padding-right: 100px
+  @media (max-width: 500px)
+    text-align: left
+    padding-right: 0
```

Comment:
You might want a little bit of padding here in case the text is wider than the device width

Reply:
There's actually a `.container` class that adds padding around the text - the paddings here are mostly to bring the .container to the right/left edges of the BrickHack logo/MLH flag.

I'll add a comment to the css

<img width="371" alt="screenshot 2018-01-08 13 02 26" src="https://user-images.githubusercontent.com/1441807/34684809-58cf5350-f474-11e7-83a1-c8e0f46c06d8.png">